Jenkinsfile

```diff
@@ -108,231 +114,388 @@ pipeline {
+                                        sh "npm pack @zowe/zosmf-for-zowe-sdk@6.31.1"
+
+                                        sh "../scripts/repackage_bundle.sh *.tgz" // Outputs a zowe-cli-package.zip
+                                        sh "mv zowe-cli-package.zip ../zowe-nodejs-sdk-${ZOWE_CLI_BUNDLE_VERSION}.zip"
+
+                                        sh "../scripts/generate_typedoc.sh ${ZOWE_CLI_BUNDLE_VERSION} ${imperativeVersion} ${zoweCliVersion}" // Outputs a zowe-node-sdk-typedoc.zip
+                                        sh "mv zowe-node-sdk-typedoc.zip ../zowe-nodejs-sdk-typedoc-${ZOWE_CLI_BUNDLE_VERSION}.zip"
+
+                                        // Remove all tgzs after bundle has been generated
+                                        sh "rm -f *.tgz"
+                                    }
+
+                                    archiveArtifacts artifacts: "zowe-nodejs-sdk*-${ZOWE_CLI_BUNDLE_VERSION}.zip"
+                                }
+                            }
+                        }
+                        /************************************************************************
+                        * STAGE
+                        * -----
+                        * Build Zowe Python SDK Bundle
+                        *
+                        * TIMEOUT
+                        * -------
+                        * 10 Minutes
+                        *
+                        * EXECUTION CONDITIONS
+                        * --------------------
+                        * - Always
+                        *
+                        * DESCRIPTION
+                        * ----------
+                        * Gets the latest version of the Zowe Python SDK package from pypi.org
+                        *
+                        * OUTPUTS
+                        * -------
+                        * A Zowe Python SDK Archive.
+                        ************************************************************************/
+                        stage('Python SDK (LTS)') {
+                            steps {
+                                timeout(time: 10, unit: 'MINUTES') {
+                                    dir("lts") {
+                                        // Download all zowe wheels into a temp folder
+                                        sh "mkdir -p temp && cd temp && pip3 download zowe"
+                                        sh "cd temp && mkdir -p licenses && cd licenses && curl -fs -o zowe_licenses_full.zip $ZOWE_LICENSE_ZIP_URL"
+
+                                        // Zip all zowe wheels into a zowe-sdk.zip
+                                        sh "cd temp && zip -r zowe-sdk.zip * && mv zowe-sdk.zip ../ && rm -rf temp"
+
+                                        // Archive the zowe Python SDK
+                                        sh "mv zowe-sdk.zip ../zowe-python-sdk-${ZOWE_CLI_BUNDLE_VERSION}.zip"

```

Comment:
I know this was from like this from before, but could we re-write some of it to use `dir("temp")` or some other not-hacky method. 😋